### ISSUE TITLE
2.8.1 Breaking Changes - Module not found: Default condition should be last one

### ISSUE THREAD

**Issue by brightiron** [alchemyplatform/alchemy-sdk-js#313]
### [REQUIRED] Environment
- Alchemy SDK version: 2.8.1
- typescript: 5.0.4
- next: 13.3.1
### [REQUIRED] Describe the problem
After updating from 2.8.0 to 2.8.1 app will no longer build. 
tsconfig.json has "moduleResolution": "node" as nextjs forces module resolution to node. 
I saw in the release notes for 2.8.1 there was a fix for the SDK previously not being compatible with bundler. It appears that has potentially broken previously compatible moduleResolutions. 

```
Module not found: Default condition should be last one
> 1 | import { Alchemy, Network } from "alchemy-sdk";
```

#### How to reproduce:
Upgrade from 2.8.0 to 2.8.1 in a nextjs environment 
Import the alchemy sdk

**Comment by wenyangliu** [comment]
same issue to me

**Comment by thebrianchen** [comment]
@brightiron Is this only on nextjs, or are you seeing in other places as well? will investigate this next week

**Comment by msio** [comment]
same error on create-react-app (react 18)

**Comment by msio** [comment]
> @brightiron Is this only on nextjs, or are you seeing in other places as well? will investigate this next week
same error on create-react-app (react 18)

**Comment by harryhexhash** [comment]
Currently, seeing this on next.js (v13.3.0). Downgrading to 2.8.0 resolves the issue.

**Comment by haeli05** [comment]
Module not found: Default condition should be last one

I solved this by manually going into node_modules/alchemy-sdk/package.json. :

Line 26: 35::
 "exports": {
    "node": {
      "types": "./dist/src/index.d.ts",
      "require": "./dist/cjs/index.js",
      "import": "./dist/cjs/index.js"
    },
    "package.json": "./package.json",
    "types": "./dist/src/index.d.ts",
    "default": "./dist/esm/index.js"
  },

Make sure "default": 
is the last line of the export.

**Comment by M-Picco** [comment]
In case the issue comes from a transitive dependency, try using `overrides` in your package file:

```
  "overrides": {
    "alchemy-sdk": "2.8.0"
  }
```

**Comment by thebrianchen** [comment]
This issue should be fixed if in `v2.8.2` and above.
